services/watson-extension/src/watson_extension/clients/platform/notifications.py

```diff
@@ -13,7 +13,18 @@
 
 class PlatformNotificationsClient(abc.ABC):
     @abc.abstractmethod
-    async def get_available_bundles(self): ...
+    async def get_available_bundles(self) -> Dict: ...
+
+    @abc.abstractmethod
```

Comment:
If we start consuming this somewhere else, might want to add a type for the bundles and groups. all good now though

Reply:
This function actually returns a list of dictionaries, updated.
As for the type, I am doing a little bit of type casting for this function outside of this function. I'm casting the bundle dictionary in a `BundleInfo` in `validate_notifications_bundle`.

I guess it'd be cleaner if `get_available_bundles` returns a list of `BundleInfo` other than a list of dictionaries, idk. But, having the type casting outside of the function in `validate_notifications_bundle` allows us to write less code, I think. 🤔

Reply:
Sure im good with less code haha. if we called the function in another route in the future, it would be easier to tell what type it is.

ig its a tradeoff, lets keep the casting :+1:

Reply:
Yeah, it'll make sense if we use this function in multiple places 👍